strr-base-web/app/utils/todoItems.ts

```diff
@@ -67,8 +67,10 @@ export const getTodoApplication = (
       newLine: '<br/>',
       boldStart: '<strong>',
       boldEnd: '</strong>',
-      linkStart: "<a href='#summary-supporting-info' class='text-blue-500 underline'>",
-      linkEnd: '</a>'
+      linkStart: "<button type='button'" +
```

Comment:
Used button for accessibility (so user can tab into it)

Reply:
So breadcrumb issue was only triggered if tabbing into Todo?

Reply:
No, originally anchor links were modifying the url (by adding the id of the element to scroll). Not sure why breadcrumb was getting updated on url change... 
I just did a different solution, which is a smooth scroll without url update / anchor link.